src/views/HomeView.vue

```diff
@@ -1,11 +1,23 @@
 <template>
-  <div class="home">
-    <TopGamesCarousel :slides="carouselSlides" />
+  <div :class="$style.home">
+    <TopGamesCarousel
+      :slides="carouselSlides"
+      :class="$style['home__games-carousel']"
+      :should-start-auto-play="true"
+    />
+    <template v-for="gameDetail in gameDetails" :key="gameDetail">
```

Comment:
Binding v-for keys to an object is REALLY not recommended. Vue needs a unique, primitive value (string or number) to efficiently track, update, and reorder elements.

Add a unique id property to each object in your gameDetails array and use that for the key (e.g., :key="gameDetail.id"). This ensures predictable and performant rendering.

```suggestion
    <template v-for="gameDetail in gameDetails" :key="gameDetail.id">
```